src/io/flutter/editor/ActiveEditorsOutlineService.java

```diff
@@ -110,19 +110,21 @@ private void updateActiveEditors() {
 
     final Set<String> newPaths = new HashSet<>();
     for (VirtualFile file : files) {
-      if (FlutterUtils.isDartFile(file)) {
+      if (FlutterUtils.isDartFile(file) && file.getCanonicalPath() != null) {
```

Comment:
wow. didn't realize we could have Dart files without an actual path. Guess it makes sense.